spec/unit/type/file/source_spec.rb

```diff
@@ -200,7 +206,7 @@
       @metadata.stubs(:mode).returns 0173
       @resource[:source_permissions] = :use
       if Puppet::Util::Platform.windows?
-        expect { @source.copy_source_values }.to raise_error("Should not have tried to use source owner/mode/group on Windows")
+        expect { @source.copy_source_values }.to raise_error("Should not have tried to use source owner/mode/group on Windows (file: my/file.pp, line: 5)")
```

Comment:
Is this always going to be line: 5? Seems like a dynamic thing that could change if code changes?

Reply:
In this context, it will, because of the stubbing that happens on lines 191-192 in the spec file.